src/commands/appSettings/AzureWebJobsStoragePromptStep.ts

```diff
@@ -5,25 +5,34 @@
 
 import { MessageItem } from 'vscode';
 import { AzureWizardPromptStep, ISubscriptionWizardContext, IWizardOptions, StorageAccountKind, StorageAccountListStep, StorageAccountPerformance, StorageAccountReplication } from 'vscode-azureextensionui';
-import { isWindows, localSettingsFileName } from '../../constants';
+import { isWindows } from '../../constants';
 import { ext } from '../../extensionVariables';
 import { localize } from '../../localize';
 import { IAzureWebJobsStorageWizardContext } from './IAzureWebJobsStorageWizardContext';
 
 export class AzureWebJobsStoragePromptStep<T extends IAzureWebJobsStorageWizardContext> extends AzureWizardPromptStep<T> {
+    private _suppressSkipForNow?: boolean;
+
+    public constructor(suppressSkipForNow?: boolean) {
+        super();
+        this._suppressSkipForNow = suppressSkipForNow;
+    }
+
     public async prompt(context: T): Promise<void> {
         const selectAccount: MessageItem = { title: localize('selectAzureAccount', 'Select storage account') };
         const useEmulator: MessageItem = { title: localize('userEmulator', 'Use local emulator') };
         const skipForNow: MessageItem = { title: localize('skipForNow', 'Skip for now') };
 
-        const message: string = localize('selectAzureWebJobsStorage', 'AzureWebJobsStorage must be set in "{0}" to debug non-HTTP triggers locally.', localSettingsFileName);
+        const message: string = localize('selectAzureWebJobsStorage', 'In order to debug, you must select a storage account for internal use by the Azure Functions runtime.');
```

Comment:
I'm a little concerned that removing the non-HTTP part might confuse users. Considering HTTP is the most popular trigger, I could see a situation where a user has been using HTTP triggers, decides to try a Timer trigger, and think that it's an error that this message has come up all of a sudden when they never needed storage to debug in the past.

Reply:
I see where you're coming from, but I feel like there's going to be some confusion no matter what and thus I prioritized making the message as simple/straightforward as possible. For example, even if I say "non-HTTP" I feel like users would wonder "What does a timer trigger have to do with a storage account?". We can change based on feedback